### ISSUE TITLE
Keyboard nav in dropdown menus broken in 6.4 RC

### ISSUE THREAD

**Issue by kball** [foundation/foundation-sites#10129]
Attempting to use keyboard nav in the dropdown menus in the visual test /dropdown-menu/left-aligned-dropdown-menu-inside-top-bar-right.html is missing the first item, appears to be unable to find it.

Possibly related, we currently have 3 failing javascript tests associated with dropdown menus.

@Owlbertz can you take a look? Thanks

**Comment by Owlbertz** [comment]
@kball, the issue is caused by the fact that the first link has no `href`-attribute and thus is not focus-able. We would either need to force a `href="#"` or to go with using `tabindex`  I guess.

**Comment by kball** [comment]
OK... we can prob enforce/require the href... can you see if the failing dropdown menu JS tests are related?

**Comment by Owlbertz** [comment]
Well as far as I can see the tests use a template that has a `href` present for each link, so it is probably another issue.

**Comment by DanielRuf** [comment]
The tests should be fine now. Does the initial issue still exist / occur?

**Comment by DanielRuf** [comment]
The visual test still has the anchor without href or tabindex. Should we use `href="#"` like in the docs or add `tabindex="0"`?

In the real world there is no anchor in a nav without a href or tabindex.

**Comment by DanielRuf** [comment]
Closing as we did not receive any further feedback and the initial issue should be resolved.
